src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -411,4 +407,59 @@ void CryptoManagerImpl::InitCertExpTime() {
   strptime("1 Jan 1970 00:00:00", "%d %b %Y %H:%M:%S", &expiration_time_);
 }
 
+bool CryptoManagerImpl::SaveModuleCertificateToFile(X509* certificate) const {
```

Comment:
Same as for PR #2217 - I suppose we should free allocated resources...

Reply:
@olohvinenko thanks, fixed in https://github.com/smartdevicelink/sdl_core/pull/2218/commits/389a64f182249a4154554788f320575a697e2092